### ISSUE TITLE
[FEATURE] Download media from replies

### ISSUE THREAD

**Issue by snwefly** [Serene-Arc/bulk-downloader-for-reddit#918]
Is there any way to grab media (gfycat/sites supported by yt-dlp) from the replies

example link https://old.reddit.com/r/reactiongifs/comments/157nt4k

one yt link in the reply.

Thanks

**Comment by Serene-Arc** [comment]
No. You can use the archive function to download the comments in a readable format and then download them if you wish with another script, but that is not currently possible.
